src/pages/User/components/PullRequests/index.js

```diff
@@ -87,9 +88,14 @@ class PullRequests extends Component {
           )
       );
 
-      const [data, userDetail] = await Promise.all(allResponses);
+      const [data, userDetail, lFMembershipStatus] = await Promise.all(allResponses);
```

Comment:
This same function right now does a lot of things. Can you extract this logic to a separate function responsible for fetching user's information only? Something like this. 

```js
const userInfo = await fetchUserInfo(username); // This should do everything under the hood and return a single object. 

... 

async function fetchUserInfo(username) {
  ...  
  const [data, detail, membershipStatus] = await Promise.all(...);
  const isOrgMember = !!membershipStatus;

  return { data, detail, isOrgMember };
} 
```